catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -24,7 +24,15 @@
     <artifactId>catalog-rest-endpoint</artifactId>
     <packaging>bundle</packaging>
     <name>DDF :: Catalog :: REST :: Endpoint</name>
+
+    <properties>
+        <pax.exam.version>4.11.0</pax.exam.version> <!-- TODO: Move to root pom file -->
```

Comment:
Are we not able to use the versions from the component-test pom? I'm not too familiar with maven property propagation.